msm/src/mvlookup.rs

```diff
@@ -361,8 +375,9 @@ pub mod prover {
                 for i in 0..domain.d1.size as usize {
                     // phi(1) = 0
                     evals.push(acc);
-                    // FIXME: only one lookup for now.
-                    acc += lookup_terms_evals_d1[i]
+                    for lte in lookup_terms_evals_d1.iter() {
+                        acc += lte[i]
```

Comment:
Do we want to assume that the starting and final terms of the accumulator are zero, or we want to be more general so we can define this "iteratively", and then only at the end it all cancels out?

Reply:
I assume it is zero atm.